artist-traits-rx/src/main/kotlin/com/uber/artist/traits/rx/KotlinApiHelper.kt

```diff
@@ -187,7 +188,7 @@ fun addRxBindingApiForSettable(type: TypeSpec.Builder, api: KotlinSettableApi, i
               artistRxConfig.processTap(this)
             }
           }
-          .addStatement("\n\t.subscribe($rxBindingMethod)")
+          .addStatement("\n\t.subscribe($rxBindingMethod)\n\t}\n")
```

Comment:
i think you'll have to change this:
```suggestion
          .addStatement("\n\t.subscribe(it)\n\t}\n")
```